### DISCUSSION TITLE
Filtering Indirectly  expanded table

### DISCUSSION THREAD

**Discussion by Dragan321** [pocketbase/pocketbase#1917] [Q&A] [answered]
So, I got  two tables one is expanded indirectly onto another like this 
`await pb.collection('narudzbe').getList(1, 50, {
                expand: 'user_id,sadrzajNarudzbe(narudzbe_id), sadrzajNarudzbe(narudzbe_id).cvijet_id ',
                '$autoCancel': false
            })`

Is it possible to filter this by  a field from indirectly expanded table  for example sadrzajNarudzbe(narudzbe_id).cvijet_id.someField coz I tried many different combinations but all of them net bad request?

**Comment by ganigeorgiev** [comment, marked as answer]
Filter and sort via indirect relation field(s) at the moment is not supported but it is in the [roadmap](https://github.com/orgs/pocketbase/projects/2) and eventually will be added sometime in the future (_I have some performance concerns and need to test it more thoroughly before enabling it_).

**Reply by Dragan321** [reply]
Thanks, I just figured out that a  join field can have multiple values if you leave max select empty so indirect join wasn't event necessary in this case.
